packages/edit-post/src/components/layout/index.js

```diff
@@ -273,6 +274,57 @@ function Layout( { initialPost } ) {
 		);
 	}
 
+	const { createSuccessNotice } = useDispatch( noticesStore );
+
+	const onActionPerformed = useCallback(
+		( actionId, items ) => {
+			switch ( actionId ) {
+				case 'move-to-trash':
+					{
+						const postType = items[ 0 ].type;
+						document.location.href = addQueryArgs( 'edit.php', {
+							post_type: postType,
+						} );
+					}
+					break;
```

Comment:
Side note: The `move-to-trash `action doesn't display a notice in the post table list view after trashing the post. We'll need to match the main "Move to Trash" button behavior.

## Screenshot
![CleanShot 2024-05-13 at 13 37 12](https://github.com/WordPress/gutenberg/assets/240569/cfc666d8-7e80-42fd-91cb-092707e695e2)